lizrd/support/code_versioning.py

```diff
@@ -25,15 +27,44 @@ def commit_pending_changes(repo: Repo):
         repo.git.commit(m="Versioning code", no_verify=True)
 
 
-def version_code(versioning_branch: str, repo_path: Optional[str] = None):
+def create_run_experiment_script(
+    experiment_config_path, experiment_branch_name, file_path
+):
+    script_text = f"#!/bin/bash\npython3 -m lizrd.grid.grid --config_path={experiment_config_path} --git_branch={experiment_branch_name} --skip_copy_code"
```

Comment:
How about using `"""` (triple quote) and making it a multiline string so that we don't have to manually look for this `\n` alternatively please make it as `...bash\n" + "python3...` for readability reasons

Reply:
also it's not `lizrd.grid.grid` anymore

Reply:
dne